src/index.js

```diff
@@ -0,0 +1,48 @@
+const env = require('./env');
+const migrations = require('./migrations');
+const refreshMatViews = require('./refresh-materialized-views');
+const log = require('loglevel');
+
+log.setDefaultLevel('info');
+
+if (!env.validateEnv()) {
+  process.exit(1);
+  return; // stop execution in tests
+}
+
+if (process.argv.length > 2 && process.argv[2] === '--usage') {
+  log.info(`
+  USAGE
+  
+  export POSTGRESQL_URL=<PostgreSQL db URL>
+  export RAPIDPRO_URL=<RapidPro instance URL>  
+  export RAPIDPRO_AUTH=<RapidPro authentication token>
+  rapidpro2pg
+  `);
+  process.exit(0);
```

Comment:
```suggestion
  process.exit(process.argv.length > 2 ? -1 : 0);
```

I would try to return a different value of 0 if there was an error in the params in cases the process return is evaluated in a script.

Reply:
This isn't an error in params, it's a `--usage` param that just lists which env variables are required. The only way to get to this is to do `--usage`. (cht-conf has this too).